### DISCUSSION TITLE
Getting Error with this plugin in moodle

### DISCUSSION THREAD

**Discussion by christy6953** [alokrai1/moodle-mod_gotomeeting#60] [Q&A]
I have installed the goto meeting plugin to my moodle server 
with the API keys generated with goto meeting accounts.
I was able to host sessions with one account...
then i have added one more goto meeting account with same procedures 
i have done before..it was perfectly working for one session 
and later it is throwing error....
i am not able to create another course with it...its 
<img width="953" alt="incorrect function error" src="https://user-images.githubusercontent.com/130046861/230547373-f2f95cf9-ff5d-47d5-b6b7-3f4491f2f115.png">
showing "incorrect function"

the steps i have followed...
*Created a test instance for moodle..
* installed goto meeting plugin to moodle
*generated API keys with the domain and gave that key to goto meeting page in moodle.
*I was able to host a session without any error with my goto meeting plugin account.

*Then i wanted to add another account …so i created a key for that itself and added that also
To my moodle as I did before.
*It was perfectly working for one session
*for the second time i wanted to host the session its throwing error “Incorrect function”
*The same error im getting from the 1st account also which i added

**Comment by alokrai1** [comment]
Hi give me some time, I will upgrade this plug-in for 4.0

Alos it would be better you can share the your gotta meeting credentials
for testing it.
